config/src/defaults.rs

```diff
@@ -111,6 +111,11 @@ pub trait Defaults {
         10
     }
 
+    /// Default period between superblocks
+    fn consensus_constants_superblock_agreement_decreasing_period(&self) -> u32 {
+        100
```

Comment:
100 superepochs are 12 hours, maybe it is too much

Reply:
agree